classes/facades/Locale.inc.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file classes/facades/Locale.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Locale
+ *
+ * @brief This facade provides access to the locale
+ */
+
+namespace PKP\facades;
```

Comment:
The interface which I implemented to handle the translations (`\Illuminate\Contracts\Translation\Translator`) has a method with this name, so it's not possible, unless I change the code structure.

My biggest motivation was getting rid of the "Facade" (personal pre-concept), but given that we're already heavily using them (e.g. `DB`) and it kind of works faster than using the `app()`, I'll keep it this way...

Reply:
@henriqueramos has also [proposed changing](https://github.com/pkp/pkp-lib/pull/7166/files#diff-2a88d2b16f09aaae268c624e13a2605dce904385c8cb3cc4ecdd20e44bbf942b) `App::make` to `app()->make`; I don't have enough depth of knowledge of the Laravel idiom to know which is better for us, but would like us to stay consistent. Could the two of you chat a bit and make a recommendation?

Reply:
There was a small discussion about it here: https://github.com/pkp/pkp-lib/pull/7166/files#r737492347

I would normally opt-in for the `App::make()` because it's less likely to cause conflicts (namespaced), but as the `app()` works slightly faster, I ended up replacing the `App::make(xxx)` by `app(xxx)`.